### ISSUE TITLE
Update repo to consume fundamental styles instead of fiori-fundamentals

### ISSUE THREAD

**Issue by jbadan** [SAP/fundamental-ngx#880]
See https://github.com/SAP/fundamental-react/pull/566

**Comment by MattL75** [comment]
Thanks @jbadan! Any detail on the issues with fundamental-styles/latest fiori fundamentals (1.5.8) that you brought up in the ticket you linked?

**Comment by jbadan** [comment]
Yeah! It looks like a change I made in SAP/fundamental#1401 messed up the alignment of the text in the button for LocalizationEditor. 

Fundamental-react hadn't upgraded our fiori-fundamentals version in a while, so we didn't notice the issue until this pr. That bug is present in fiori-fundamentals current version as well as fundamental-styles, but I opened a pr to fix it. So totally up to you if you want to wait to switch until the fix goes in. 

https://github.com/SAP/fundamental-styles/pull/35
